CONTRIBUTING.md

```diff
@@ -190,11 +190,63 @@ Changing appearance or theme of the website from Docusaurus defaults can be done
 
 - Make sure styling changes work in both [light and dark modes](https://docusaurus.io/docs/styling-layout#dark-mode).
 
+### Package Management
+
+The website uses [pnpm](https://pnpm.io/) as a package manager. This is the same package manager used to build [Recon](https://github.com/apache/ozone/tree/master/hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web). Basic knowledge required to maintain the website's dependencies is outlined here. See [pnpm docs](https://pnpm.io/pnpm-cli) for complete usage.
+
+#### Relevant Files
+
+- **package.json**
+
+This file contains version guidelines for all the top level dependencies required to build the website. This file may be updated manually to adjust which versions are installed, or automatically when commands like `pnpm update` are run.
+
```

Comment:
What do you mean by this?

Reply:
So package.json can have different information other than just the dependencies. For ex in the current website package.json we see various metadata like:
- [specifying the package manager and version](https://github.com/apache/ozone-site/blob/4c95c8a2925fd19534c42f36593cc52fcbbc2323/package.json#L2)
- [project name](https://github.com/apache/ozone-site/blob/4c95c8a2925fd19534c42f36593cc52fcbbc2323/package.json#L3C4-L3C8)
etc.

Also it has a [scripts section](https://github.com/apache/ozone-site/blob/4c95c8a2925fd19534c42f36593cc52fcbbc2323/package.json#L6C4-L6C11) which specifies the exact command to run for the given aliases. For example `pnpm start` will run `docusaurus start`.

The package manager essentially checks this scripts section and then treats these as the equivalent of aliases for a command.
This feature is helpful when you want a long command to run, say as a part of some CI build, or to ease the dev process, you can add a new command and reference the actual command you want to run - then you can call it as `pnpm <new command name>`

Reply:
I think this is outside the scope of what I was aiming to cover. Basically I wanted to add just enough info that people could confidently do package updates with just this guide since most are probably not familiar with pnpm. That's why I put this sentence at the beginning:
> Basic knowledge required to maintain the website's dependencies is outlined here. See [pnpm docs](https://pnpm.io/pnpm-cli) for complete usage.